AkkaHotSwap/src/main/java/org/ameya/App.java

```diff
@@ -18,6 +18,9 @@ public static void main(String args[]) throws  Exception{
         pingPongActor.tell(Messages.START,null);
         Thread.sleep(10000);
 
+        System.out.println("this line has been added by other user who is not part of codacy.To test whether this is flagged in the PR");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [System.out.println is used](https://www.codacy.com/app/ameya-kulkarni/akka-goodies/file/1684666204/issues/source?bid=3041792&fileBranchId=3041863#l21)